tests/utils.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import json
+
+FIXTURES_DIR = 'tests/fixtures'
```

Comment:
This is a bit fragile, and might fail depending on from which directory you run the unit tests. There's a more robust way of handling this:

1. Make sure the json files are in `MANIFEST.in`.
2. Make sure `setup.py` has the line `include_package_data=True` (it has).
3. You can now read the json content from using `pkg_resources.resource_filename` **no matter where it was installed**.

For 1 & 2, see [this reference](http://python-packaging.readthedocs.io/en/latest/non-code-files.html). For 3, see [this reference](http://peak.telecommunity.com/DevCenter/PythonEggs#accessing-package-resources).

Reply:
You could also just have the fixtures be defined as python data structures and `json.dumps` them if/when needed